app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
```

Comment:
Please add your styling by following the given login to match [the wireframes from the Sneak Peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md#:~:text=For%20this%20project%20you%20will%20have%20full%20freedom%20in%20terms%20of%20visual%20design%20but%20you%20will%20need%20to%20keep%20the%20following%20wireframes%3A) and you can personalize your styling it has to generally obey the wireframe. Also and styling to your `Sign Up` page too.

| Your Design | Wireframe Design |
|-------------|---------------|
| <img width="225" alt="image" src="https://user-images.githubusercontent.com/34578631/174040081-bcd79bab-15f0-4a45-ab6d-c723224aa8a0.png"> <img width="233" alt="image" src="https://user-images.githubusercontent.com/34578631/174040142-876e5b81-6a7a-4dbf-b588-2e12d76c4e98.png"> | <img width="731" alt="image" src="https://user-images.githubusercontent.com/34578631/174040235-4d0ff0ea-bb3a-422b-ab73-91da4560358c.png"> |